src/api/utility-types.md

```diff
@@ -1,14 +1,14 @@
-# Utility Types {#utility-types}
+# Tipos de Utilitário de TypeScript {#utility-types}
```

Comment:
First of all, thank you for the multiple contributions so far, @ogilvieira 😁

Can we keep this title as "Tipos de Utilitário" only? In that way we'll not create words where they didn't exist before.